src/main/java/rife/database/DbStatement.java

```diff
@@ -223,13 +223,12 @@ protected long startTrace() {
 
     protected void outputTrace(long start, String sql) {
         if (start != 0) {
-            StringBuilder output = new StringBuilder();
 
-            output.append(System.currentTimeMillis() - start);
-            output.append("ms : ");
-            output.append(sql);
+            String output = (System.currentTimeMillis() - start) +
```

Comment:
It's weird to me that you've been replacing string concatenation in other pull requests with string builders and now you're replacing a string builder with string concatenation?

Reply:
No concatenation in a loop, that creates too many new objects. In this case, creating a StringBuilder is a waste of resources., as it is never modified past the original creation.